chesleybrown.md

```diff
@@ -1,83 +1,46 @@
+> Founder of [RentChek](https://rentchek.com). Lead Software Engineer at [InVision](https://www.invisionapp.com). Coding enthusiast. Husband & Father.
 
 - [chesleybrown.ca](http://chesleybrown.ca)
 - [me@chesleybrown.ca](mailto:me@chesleybrown.ca)
 - `709-697-5407`
 
-## Education
-
-- **College of the North Atlantic** - _[cna.nl.ca](http://www.cna.nl.ca)_
-
-	Website Administrator - _Class of 2007_
-	> Graduated with a _4.0 GPA_
-
 
 ## Skills
 
-- **Javascript** - _Proficient_
-
-	[NodeJS](http://nodejs.org), [AngularJS](https://angularjs.org), [jQuery](http://jquery.com), [GruntJS](http://gruntjs.com)
-
-- **HTML & CSS** - _Proficient_
-
-	[Sass](http://sass-lang.com), [Responsive Web Design](http://en.wikipedia.org/wiki/Responsive_web_design)
-
-- **Package Management** - _Proficient_
-
-	[npm](https://www.npmjs.org), [bower](http://bower.io), [Composer](https://getcomposer.org)
-
-- **PHP** - _Proficient_
-
-	[Zend Framework](http://framework.zend.com), [CakePHP](http://cakephp.org)
-
-- **Python** - _Competent_
-
-	[Django](https://www.djangoproject.com)
-
-- **Source Control** - _Proficient_
-
-	[Git](http://git-scm.com), [SVN](http://subversion.apache.org), [CVS](http://www.nongnu.org/cvs/)
-
-- **Objective-C** - _Some Experience_
-
-	[iOS](https://www.apple.com/ca/ios/), [Xcode](https://developer.apple.com/xcode/)
-
-- **Databases** - _Competent_
-
-	[MySql](http://www.mysql.com), [Postgres](http://www.postgresql.org), [MongoDB](http://mongodb.org), [Firebase](http://firebase.com)
-
-- **Unit & Functional Testing** - _Proficient_
-
-	[Mocha](http://mochajs.org), [JasmineBD](http://jasmine.github.io), [Selenium](http://docs.seleniumhq.org), [phpunit](http://phpunit.de)
-
-- **Servers** - _Competent_
-
-	[Ubuntu](http://www.ubuntu.com/download/server), [nginx](http://nginx.org), [Apache](https://httpd.apache.org)
-
-- **Infrastructure & DevOps** - _Proficient_
-
-	[Docker](http://docker.io), [Puppet](http://puppetlabs.com), [Vagrant](http://www.vagrantup.com), [Continuous Integration](http://en.wikipedia.org/wiki/Continuous_integration), [Heroku](http://heroku.com), [Modulus](http://modulus.io), [12 Factor](http://12factor.net)
-
-- **eLearning** - _Competent_
-
-	[SCORM Engine](http://scorm.com/engine), [SCORM Cloud](http://scorm.com/cloud), [SCORM 1.2/2004](http://en.wikipedia.org/wiki/Sharable_Content_Object_Reference_Model), [AICC](http://en.wikipedia.org/wiki/Aviation_Industry_Computer-Based_Training_Committee), [TinCan API](http://tincanapi.com)
-
-- **Other Skills & Traits**
-
-	Project Management, Self-initiative, Team Collaboration, Team Management, Project Planning, Forward Thinking, Knowledge Transfer, Documenting, Technical Scoping, Research & Development
+- Proficient in Golang, Javascript (TypeScript, React, Angular), HTML, CSS (Sass, Responsive Web Design), Git, Bash, Docker, Continuous Integration, Debuggers (Browser, IDE), and Unit & Functional Testing
```

Comment:
```suggestion
- Proficient in Golang, Javascript (TypeScript, React, Angular), HTML, CSS (Tailwind, Sass, Responsive Web Design), Git, Bash, Docker, Continuous Integration, Debuggers (Browser, IDE), and Unit & Functional Testing
```

Tailwind is something you could pick up very easily and is the new trend recruiters are looking for these days

Reply:
Ah, good catch. I actually have touched on it because we do use it in Freehand. I'll add it in. 👍🏻